org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/impl/XtextBuilder.java

```diff
@@ -85,6 +93,58 @@
 	public IResourceSetProvider getResourceSetProvider() {
 		return resourceSetProvider;
 	}
+
+	/**
+	 * 
+	 * @since 2.16
+	 */
+	@Singleton
+	@Beta
+	static class BuilderPreferences {
```

Comment:
Please make this public (but still `@Beta`) so others can customize it

Reply:
done